### ISSUE TITLE
How to extract items from list

### ISSUE THREAD

**Issue by dibyadip-das** [gadgetchnnel/lovelace-home-feed-card#35]
Here is a calendar config I want to add to my feed:

![image](https://user-images.githubusercontent.com/47345141/76158640-dd2f8600-613d-11ea-8b34-8548f57a6d16.png)

Now while configuring I am unable to fetch the items under all_tasks:
![image](https://user-images.githubusercontent.com/47345141/76158648-0cde8e00-613e-11ea-88fd-f23398b6788f.png)

Can you please help me resolve this issue?

**Comment by gadgetchnnel** [comment]
I have just released a new beta version (0.4.1b1) which should allow this to work (previously it would only work if the attribute referred to by **list_attribute** contained a list of objects, rather than strings. The new version allows this by treating them as as objects like {value:"string value"} and so can be referred to in content_template as {{value}}, for example:

```
type: 'custom:home-feed-card'
title: Home Feed
show_empty: false
entities:
  - entity: calendar.alexa_shopping_list
    multiple_items: true
    list_attribute: all_tasks
    content_template: '{{value}}'
    icon: 'mdi:shopping'
```

**Comment by dibyadip-das** [comment]
@gadgetchnnel Thank you for making such a great effort. How can I install it? I can see the version till 0.4.0 only. I even tried to install the dev version. Still didn't work.

![image](https://user-images.githubusercontent.com/47345141/77230915-c0736380-6bbd-11ea-80f9-c98d8f07cf5d.png)

**Comment by gadgetchnnel** [comment]
You need to enable beta versions for the plugin in HACS. To do this, click on the 3 dots menu in the corner (to the right of the "Lovelace Home Feed Card" title):
<img width="275" alt="Screenshot 2020-03-21 at 16 36 01" src="https://user-images.githubusercontent.com/2099542/77231336-1e8a5180-6b92-11ea-9055-6a1824ca7f98.png">
and then select "Show beta" from the menu:
<img width="199" alt="Screenshot 2020-03-21 at 16 36 10" src="https://user-images.githubusercontent.com/2099542/77231338-20ecab80-6b92-11ea-8a3b-54b6e74baf4c.png">

Using the **dev** version should also have worked, but can you please try with the beta version and also try a refresh of the page in the browser as this is sometimes needed.

**Comment by dibyadip-das** [comment]
Wow, it worked. But if I add 2nd Calendar that doesn't any items yet then the whole feed gets blank

Here is the config:
![image](https://user-images.githubusercontent.com/47345141/77232318-4398b780-6bc6-11ea-9230-e5ca8a22bb7c.png)

Output
![image](https://user-images.githubusercontent.com/47345141/77232377-b144e380-6bc6-11ea-96cd-e36134001c87.png)

**Comment by gadgetchnnel** [comment]
@dibyadip-das The issue was that those calendar entities with no tasks have a null **all_tasks** attribute. I've fixed this in the latest beta, 0.4.1b3, so that it treats it as an empty list and doesn't cause it to fail to load.

**Comment by dibyadip-das** [comment]
Thank you so much @gadgetchnnel for all your hard work. The issues are now fixed. I can see empty card now 👍
